src/js/mods/modloader.js

```diff
@@ -187,16 +198,25 @@ export class ModLoader {
             this.modLoadQueue.push({
                 modClass,
                 meta,
+                path,
```

Comment:
why are we tracking path and not just whether it's disabled? also I get why `let path` is isolated from the mod itself but the mod already can do so much even if it will get "disabled"

Reply:
We now, are tracking it because we use it for the metadata cache.